sensitive_data_filter.gemspec

```diff
@@ -0,0 +1,37 @@
+# coding: utf-8
+# frozen_string_literal: true
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'sensitive_data_filter/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'sensitive_data_filter'
+  spec.version       = SensitiveDataFilter::VERSION
+  spec.authors       = ['Alessandro Berardi']
+  spec.email         = ['berardialessandro@gmail.com']
```

Comment:
There was some comment that if the email has support@travellink.com.au we all end up with ownership when this is published. Worth investigation.